person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @id = rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
**[OPTIONAL]**

- Hi :clap: to improve the readability of your code, instead of having to add the `public` to indicate the 
`def can_use_service`, kindly move it up above the `private`. see:

```

def can_use_services?
    of_age? || @parent_permission
  end

  private

  def of_age?
    @age >= 18
  end
```

Reply:
thank you for this @zieeco